api/v1alpha/example_client.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""
+Copyright 2016 Author Josef Boudnik - josef.boudnik@seznam.cz  
```

Comment:
Would you be open to assigning copyright to the generic "the rkt Authors" as we use for the rest of the project?